Dodal ključne besede iz moje magistrske iz področja teorije izračunlj…

Andraz1999 opened the pull request:
…ivosti

Pozdravljeni,

sem Andraž Pustoslemšek, študent na FMF, in za potrebe moje magistrke, vas je moj mentor profesor Simpson vprašal za slovenske prevode nekaterih pojmov iz teorije izračunljivosti.
Najprej bi se vam zahvalil za vašo pomoč.

V vašem mejlu ste tudi napisali, da naj naredim pull request, ko bodo prevodi fiskirani. V dictionary.md sem dodal vse svoje ključne besede, če tudi mogoče vse ne spadajo v področje teorijo tipov. Posebej pozornost zahteva pojem "pravilno zaporedje" (proper sequence), ki je definirano kot zaporedje, ki je Martin-Löf naključno relativno na neko izračunjivo mero Cantorjevega prostora.
Izraz sem prevedel sam iz originalnega ruskega dela "THE COMPLEXITY OF FINITE OBJECTS AND
THE DEVELOPMENT OF THE CONCEPTS
OF INFORMATION AND RANDOMNESS BY
MEANS OF THE THEORY OF ALGORITHMS"
A.K. Zvonkin in L.A. Levin.
V njunem delu uporabljata izraz правильная последовательность. (Rusko sicer ne znam, znam pa brati cirilico).
Izraz "pravilno zaporedje" zato morda ni široko sprejet pojem.

Še enkrat se vam zahvaljujem za prevode in vas lepo pozdravljam.

Lep pozdrav
Andraž Pustoslemšek

Andraz1999 committed:
Dodal ključne besede iz moje magistrske iz področja teorije izračunljivosti

Pozdravljeni,

sem Andraž Pustoslemšek, študent na FMF, in za potrebe moje magistrke,
vas je moj mentor profesor Simpson vprašal za slovenske prevode
nekaterih pojmov iz teorije izračunljivosti.
Najprej bi se vam zahvalil za vašo pomoč.

V vašem mejlu ste tudi napisali, da naj naredim pull request,
ko bodo prevodi fiskirani. V dictionary.md sem dodal vse svoje ključne besede,
če tudi mogoče vse ne spadajo v področje teorijo tipov.
Posebej pozornost zahteva pojem "pravilno zaporedje" (proper sequence),
ki je definirano kot zaporedje, ki je Martin-Löf naključno relativno
na neko izračunjivo mero Cantorjevega prostora.
Izraz sem prevedel sam iz originalnega ruskega dela
"THE COMPLEXITY OF FINITE OBJECTS AND
THE DEVELOPMENT OF THE CONCEPTS
OF INFORMATION AND RANDOMNESS BY
MEANS OF THE THEORY OF ALGORITHMS"
A.K. Zvonkin in L.A. Levin.
V njunem delu uporabljata izraz правильная последовательность.
(Rusko sicer ne znam, znam pa brati cirilico).
Izraz "pravilno zaporedje" zato morda ni široko sprejet pojem.

Še enkrat se vam zahvaljujem za prevode in vas lepo pozdravljam.

Lep pozdrav
Andraž Pustoslemšek

anjapetkovic reviewed (changes requested):
Hvala za PR!
Super, a bi lahko prosila samo, da nove besede umestiš med obstoječe v abecednem vrstnem redu?

Andraz1999 committed:
dodal nove besede v abecednem vrstnem redu

Uporabil sem sort.py, da sem umestil nove besede tako, da sem v sort.py spremenil encoding na utf-8 in dodal vrstico, da sortira po malih črkah. (spremembe sort.py nisem pushal)

anjapetkovic reviewed (approved):
Hvala, izgleda super.
